website/docs/r/iothub.html.markdown

```diff
@@ -188,6 +190,16 @@ An `endpoint` block supports the following:
 
 ---
 
+A `identity` block supports the following:
+
+* `type` - (Required) The type of Managed Identity which should be assigned to the Iot Hub. Possible values are `SystemAssigned`, `UserAssigned` and `SystemAssigned, UserAssigned`.
+
+* `identity_ids` - (Optional) A list of User Managed Identity ID's which should be assigned to the Iot Hub.
+
+~> **NOTE:** This is required when `type` is set to `UserAssigned`.
```

Comment:
this is also required when it's set to SystemAssigned/UserAssigned:

```suggestion
```

Reply:
Updated